proto/forch_configuration.proto

```diff
@@ -165,6 +168,28 @@ message EventClientConfig {
  * Varz interface section in Forch configuration
  */
 message VarzInterfaceConfig {
-  // port number to curl for varz list
+  // port number for forch varz
   int32 varz_port = 1;
 }
+
+/*
+ * Target to scrape metrics from
```

Comment:
it's a target for a port proxy, has nothing to do with metrics -- names/comments here and below should reflect that.

Reply:
Addressed.